workspaces/variables.md

```diff
@@ -42,25 +54,29 @@ env | grep CODER_
         <td>The memory limit given to your workspace in GB</td>
     </tr>
     <tr>
-        <td><code>CODER_IMAGE_TAG</code></td>
-        <td>The image tag used to create your workspace</td>
+        <td><code>CODER_AGENT_TOKEN</code></td>
```

Comment:
IMHO this one doesn't need to be documented (or should only be documented as "for internal use") as this is an implementation detail and not something that people should rely on as being part of API

Reply:
got it!